test/reachsets.jl

```diff
@@ -101,5 +101,36 @@ end
     @test tspan(c) == tspan(d) == 0..1
     @test isequivalent(set(overapproximate(c, Zonotope)), set(R))
     @test isequivalent(set(overapproximate(d, Zonotope)), set(R))
+                    
+end
 
+@testset "Taylor model reach-sets with non-float coefficients" begin
+    
+    using TaylorModels:Taylor1
```

Comment:
by convention, the imports are added at the header of the file (see other files for examples)